src/common.js

```diff
@@ -4,11 +4,23 @@
 
 var os = require('os');
 var fs = require('fs');
+var path = require('path');
 var glob = require('glob');
 var shell = require('..');
 
 var DEFAULT_ERROR_CODE = 1;
 var shellMethods = Object.create(shell);
+var PATH = 'PATH';
+
+// windows calls it's path 'Path' usually, but this is not guaranteed.
```

Comment:
I believe we can rely on `process.env` to have case-insensitive keys, so `process.env.PATH` should always work.

`process.env` isn't a real JS object. See atom/atom#11302 as an example of this.